webpack/assets/javascripts/react_app/components/Layout/components/TaxonomyDropdown.js

```diff
@@ -1,101 +1,97 @@
+        <li>
+          <a
+            className={`${taxonomyType}s_clear`}
+            href={anyTaxonomyURL}
+            onClick={() => {
+              changeTaxonomy({ title: anyTaxonomyText });
+            }}
+          >
+            {__(anyTaxonomyText)}
+          </a>
+        </li>
+        <li>
+          <a className={taxonomyType} href={manageTaxonomyURL}>
+            {taxonomyType === 'organization'
+              ? __('Manage Organizations')
+              : __('Manage Locations')}
+          </a>
+        </li>
+        <li className="divider" />
+        {taxonomies.length > 6 && (
           <li>
+            <input
+              type="text"
+              className="form-control taxonomy_search"
+              id={`search_taxonomy_${taxonomyType}`}
+              placeholder="Filter ..."
+              onChange={e => {
+                setSearch(e.target.value);
+              }}
+            />
+          </li>
+        )}
+        {filteredTaxonomies().map((taxonomy, i) => (
+          <li key={i}>
             <a
-              className={`${taxonomyType.toLowerCase()}s_clear`}
-              href={anyTaxonomyURL}
+              className={`${taxonomyType}_menuitem`}
+              id={`select_taxonomy_${taxonomy.title}`}
+              href={taxonomy.href}
               onClick={() => {
-                changeTaxonomy({ title: anyTaxonomyText });
+                changeTaxonomy({ title: taxonomy.title, id: taxonomy.id });
               }}
             >
-              {__(anyTaxonomyText)}
+              {taxonomy.title}
             </a>
           </li>
-          <li>
-            <a className={taxonomyType.toLowerCase()} href={manageTaxonomyURL}>
-              {__(manageTaxonomyText)}
-            </a>
-          </li>
-          <li className="divider" />
-          {taxonomies.length > 6 && (
-            <li>
-              <input
-                type="text"
-                className="form-control taxonomy_search"
-                id={`search_taxonomy_${taxonomyType.toLowerCase()}`}
-                placeholder="Filter ..."
-                onChange={e => {
-                  this.setState({ searchValue: e.target.value });
-                }}
-              />
-            </li>
-          )}
-          {filteredTaxonomies().map((taxonomy, i) => (
-            <li key={i}>
-              <a
-                className={`${taxonomyType.toLowerCase()}_menuitem`}
-                id={`select_taxonomy_${taxonomy.title}`}
-                href={taxonomy.href}
-                onClick={() => {
-                  changeTaxonomy({ title: taxonomy.title, id: taxonomy.id });
-                }}
-              >
-                {taxonomy.title}
-              </a>
-            </li>
-          ))}
-        </ul>
-      </NavItem>
-    );
-  }
-}
+        ))}
+      </ul>
+    </NavItem>
+  );
+};
 
 TaxonomyDropdown.propTypes = {
   taxonomyType: PropTypes.string.isRequired,
```

Comment:
Is there an enum type to ensure it's either `organization` or `location`?

Reply:
added